.github/workflows/tox.yml

```diff
@@ -166,10 +166,15 @@ jobs:
 
       - name: Setup conda
         if: ${{ matrix.conda == 'true' }}
-        uses: s-weigand/setup-conda@v1
+        uses: mamba-org/provision-with-micromamba@v15
         with:
-          update-conda: true
-          python-version: ${{ matrix.python_version }}
+          environment-file: false
+          environment-name: 'test'
+          extra-specs: |
+            python=${{ matrix.python_version }}
+          channels: defaults
```

Comment:
Should we change this to `conda-forge`? I guess this will use the Anaconda `defaults` channel for installing Python, but people can then specify `conda-forge` in their tox-conda config?

Reply:
Good point; it might just be better to use `conda-forge` by default.